docs/ref/building.rst

```diff
@@ -179,13 +234,15 @@ can be linked into an application::
   add_executbale(my-program main.c)
   target_link_libraries(my-program PRIVATE amongoc::amongoc)
 
-Dependency Imports
-******************
-
 By default, the |amongoc| CMake package will attempt to import dependencies
 using :cmake:command:`find_dependency <command:find_dependency>`. This import
 can be disabled by changing :cmake:variable:`AMONGOC_FIND_DEPENDENCIES`.
 
+**If you build** |amongoc| using vcpkg_ (the default) it is highly recommended
+to use vcpkg in your own project to install |amongoc|'s dependencies, as it is
```

Comment:
Consider adding a sample project with instructions to install amongoc and building a consumer with vcpkg. Example: https://gist.github.com/kevinAlbs/636fb109747ee4faba16def81c578a93 That may help give a fast-path to testing the driver.